util/chunk/chunk_test.go

```diff
@@ -104,62 +101,64 @@ func (s *testChunkSuite) TestChunk(c *check.C) {
 	chk.AppendSet(5, setVal)
 
 	row := chk.GetRow(0)
-	c.Assert(row.GetFloat32(0), check.Equals, f32Val)
-	c.Assert(row.GetTime(2).Compare(tVal), check.Equals, 0)
-	// fsp is no longer maintained in chunk
```

Comment:
Why we need to remove this comment?

Reply:
I added this in the API docstring. This is not relevant in the test